beacon-chain/blockchain/process_block_test.go

```diff
@@ -2804,7 +2804,7 @@ func TestProcessLightClientUpdate(t *testing.T) {
 	require.NoError(t, s.cfg.BeaconDB.SaveState(ctx, headState, [32]byte{1, 2}))
 	require.NoError(t, s.cfg.BeaconDB.SaveHeadBlockRoot(ctx, [32]byte{1, 2}))
 
-	for testVersion := version.Altair; testVersion <= version.Electra; testVersion++ {
+	for testVersion := version.Altair; testVersion <= version.Fulu; testVersion++ {
```

Comment:
Not sure if this would work, but it could be interesting


```suggestion
	for testVersion range version.All()[1:] {
```

Something where you automatically reference all versions, except phase0, so that a new version added will be tested in the future.